File: C language/nested_if.c
/*
A number greater than zero will be passed as input.
Fill in the code in the below program so the output is as per the conditions mentioned below.

- If the input number is even and greater than or equal to 1000 print largeeven
- If the input number is even and less than 1000 print smalleven
- If the input number is odd and greater than or equal to 999 print largeodd
- If the input number is odd and less than 999 print smallodd
*/


#include <stdio.h>    

int main() {
    int number;
    scanf("%d", &number);

    if (number % 2 == 0) {
        if (number >= 1000) {
            printf("largeeven");
        } else {
            printf("smalleven");
        }
    } else {
        if (number >= 999) {
            printf("largeodd");
        } else {
            printf("smallodd");
        }
    }
    return 0;
}

File: Python/nested_if.py
# A number is passed as the input to the program. The program must print the output based on the 
# conditions given below.
# - If the input number is odd and less than or equal to 1000, 
# then the output must be double the number.
# - If the input number is odd and more than 1000 but less than or equal to 10000, 
# then the output must be 200 less than the given number.
# - If the input number is odd and more than 10000, 
# then the output must be 5000 less than the given number.
# - Else the input number is printed as such.



num = int(input())
if num%2 == 1:
    if num <= 1000:
        print (num*2)
    elif num <= 10000:
        print(num-200)
    else:
        print (num-5000)
else:
    print (num)
 